fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -215,6 +215,46 @@ The `keyword.strings` and `title.strings` are standard `.strings` file you alrea
 - These `.strings` files **MUST** be utf-8 (UTF-8) or utf-16 encoded (UTF-16 BE with BOM). They also must begin with an empty line. If you are having trouble see [issue #1740](https://github.com/fastlane/fastlane/issues/1740)
 - You **MUST** provide a background if you want titles. _frameit_ will not add the tiles if a background is not specified.
 
+### Screenshot orientation
+
+By default _frameit_ adds a frame to your screenshot based on an orientation you took it. For a portrait (vertical orientation) it is going to add portrait frame and for a landscape (horizontal orientation) - landscape left.
+
+If you'd like to get a frame in a landscape right for your horizontal screenshots, you need to add `force_landscaperight` at the end of your screenshot's filename. 
+
+### `force_orientation_block`
+
+In a case you don't want to rename your screenshots or default behaviour does not meet your requirements, there's a way to customise it. In order to do so, you need to set `force_orientation_block` parameter.
+
+### Examples
+
+```ruby
+frameit(
+  path: "./fastlane/screenshots",
+  force_orientation_block: proc do |filename|
+    case filename
+      when "1Login" 
```

Comment:
This may be nitpicky but we may want to rework this example as I don't think it will actually work if copy pasted 😇 `filename` returns something like "iPad Pro (12.9-inch)-01LoginScreen" so this example may be confusing if copy pasted 🤔

Reply:
Actually it depends on the way the users generate the screenshots. The way I was testing the parameter was just running a sample application, hit `cmd + s` and move the screenshot to the proper catalog. Then I renamed it as I wanted. However I can imagine that most common scenario to generate screenshots is `snapshot` tool. In that case, indeed would be great provide better context to our example 🙂 

```ruby
frameit(
  path: "./fastlane/screenshots",
  force_orientation_block: proc do |filename|
    case filename
      when "iPad Pro (12.9-inch)-01LoginScreen" 
        :landscape_right
      when "iPhone 6 Plus-01LoginScreen"
        : portrait
      # and so on
    end
  end
)
```
WDYT @joshdholtz ? 🤔

Reply:
Better. Maybe we can also add an example that matches only part of the filename to decide?

Reply:
Oh right - ignore that and just keep these two examples.